wordle-web/src/router/index.ts

```diff
@@ -0,0 +1,28 @@
+import { createRouter, createWebHistory } from 'vue-router'
+import HomeView from '../views/HomeView.vue'
+
+const router = createRouter({
+  history: createWebHistory(import.meta.env.BASE_URL),
+  routes: [
+    {
+      path: '/',
+      name: 'home',
+      component: HomeView
+    },
+    {
+      path: '/about',
+      name: 'about',
+      // route level code-splitting
+      // this generates a separate chunk (About.[hash].js) for this route
+      // which is lazy-loaded when the route is visited.
+      component: () => import('../views/AboutView.vue')
+    },
+    {
+        path: '/game',
+        name: 'game',
+        component: () => import('../views/game.vue')
```

Comment:
Grant had mentioned in class that the format used for the home page route is better. ie importing the route at the top and referencing it later as opposed to importing in the same line. 

So if you replace the code on line 23 with: `component: GameView`
And then add after line 2:  `import GameView from '../views/GameView.vue'`